src/app/components/board/board.component.ts

```diff
@@ -14,19 +15,23 @@ export class BoardComponent implements OnInit {
   @Input() board: Board;
   @Output() boardDeleted = new EventEmitter<boolean>();
 
+  trackByUserId = trackById;
+
   public boardId: string;
+  public users: User[];
 
-  constructor(private boardsService: BoardsService) { }
+  constructor(private boardsService: BoardsService) {}
 
   ngOnInit() {
+    if (this.board.users) {
```

Comment:
Firstly, you should check `this.boards` existing. And it's a good practice to use technique of "early return", for example:
```
if (!this.board.users) {
  return;
}
```